packages/solo/package.json

```diff
@@ -33,7 +33,6 @@
     "@agoric/telemetry": "^0.2.0",
     "@agoric/vats": "^0.9.0",
     "@agoric/wallet": "^0.13.0",
-    "@agoric/xsnap": "^0.12.0",
```

Comment:
Why?

Reply:
This is a drive-by. There is no motivation except that xsnap is not uttered anywhere in the solo package, so it appears to be extraneous.

Reply:
yeah I think swingset has that dependency for you